Source/WebKit/UIProcess/wpe/AcceleratedBackingStoreDMABuf.cpp

```diff
@@ -219,6 +234,14 @@ RendererBufferFormat AcceleratedBackingStoreDMABuf::bufferFormat() const
         }
         format.usage = DMABufRendererBufferFormat::Usage::Rendering;
     }
+#if OS(ANDROID)
+    else if (WPE_IS_BUFFER_ANDROID(buffer)) {
+        auto* bufferAndroid = WPE_BUFFER_ANDROID(buffer);
+        // XXX: Given that we transform the AHardwareBuffer format into a DRM fourcc, we can abuse ::Type::DMABuf here.
+        format.type = RendererBufferFormat::Type::DMABuf;
```

Comment:
I'd name it `HardwareBuffer` instead in a different pre-requisite commit.

Reply:
I've added a new entry to the `enum`, this is used only to describe the buffers being used in the `webkit://gpu` page and it seems worth it to be precise with what we show there.